tests/cellfinder_core/test_unit/test_cellfinder.py

```diff
@@ -6,43 +6,74 @@
 import pooch
 import pytest
 
-from brainglobe_workflows.cellfinder_core.cellfinder import (
-    CellfinderConfig,
-    add_signal_and_background_files,
-    read_cellfinder_config,
-    run_workflow_from_cellfinder_run,
-    setup_workflow,
-)
-from brainglobe_workflows.cellfinder_core.cellfinder import setup as setup_full
 from brainglobe_workflows.utils import setup_logger
 
 
-@pytest.fixture()
-def default_input_config_cellfinder() -> Path:
-    """Return path to default input config for cellfinder workflow
+@pytest.fixture(scope="session")
+def cellfinder_GIN_data() -> dict:
+    """Return the URL and hash to the GIN repository with the input data
 
     Returns
     -------
-    Path
-        Path to default input config
-
+    dict
+        URL and hash of the GIN repository with the cellfinder test data
     """
-    from brainglobe_workflows.utils import DEFAULT_JSON_CONFIG_PATH_CELLFINDER
+    return {
+        "url": "https://gin.g-node.org/BrainGlobe/test-data/raw/master/cellfinder/cellfinder-test-data.zip",
+        "hash": "b0ef53b1530e4fa3128fcc0a752d0751909eab129d701f384fc0ea5f138c5914",  # noqa
+    }
+
+
+@pytest.fixture()
+def config_local(cellfinder_GIN_data, default_input_config_cellfinder):
+    """ """
+
+    from brainglobe_workflows.cellfinder_core.cellfinder_core import (
+        CellfinderConfig,
+    )
 
-    return DEFAULT_JSON_CONFIG_PATH_CELLFINDER
+    # read default config as dict
+    # as dict because some paths are computed derived from input_data_dir
+    with open(default_input_config_cellfinder) as cfg:
+        config_dict = json.load(cfg)
+
+    # modify location of data?
+    # - remove url
+    # - remove data hash
+    # - add input_data_dir
```

Comment:
Are these still needed or now out-of-date?

Reply:
I rephrased to make it clear that it's a comment for the next few lines